Lib/colorsys.py

```diff
@@ -11,60 +11,79 @@
 
 Supported color systems:
 RGB: Red, Green, Blue components
-YIQ: Luminance, Chrominance (used by composite video signals)
+YIQ: Luminance, Chrominance (used by NTSC composite video signals)
+YUV: Luminance, Chrominance (used by PAL composite video signals)
 HLS: Hue, Luminance, Saturation
 HSV: Hue, Saturation, Value
 """
 
 # References:
-# http://en.wikipedia.org/wiki/YIQ
-# http://en.wikipedia.org/wiki/HLS_color_space
-# http://en.wikipedia.org/wiki/HSV_color_space
-
-__all__ = ["rgb_to_yiq","yiq_to_rgb","rgb_to_hls","hls_to_rgb",
-           "rgb_to_hsv","hsv_to_rgb"]
+# https://en.wikipedia.org/wiki/YIQ
+# https://en.wikipedia.org/wiki/YUV
+# https://en.wikipedia.org/wiki/HLS_color_space
+# https://en.wikipedia.org/wiki/HSV_color_space
+
+__all__ = [
+    "rgb_to_yiq", "yiq_to_rgb",
+    "rgb_to_yuv", "yuv_to_rgb",
+    "rgb_to_hls", "hls_to_rgb",
+    "rgb_to_hsv", "hsv_to_rgb"
+    ]
 
 # Some floating point constants
 
 ONE_THIRD = 1.0/3.0
 ONE_SIXTH = 1.0/6.0
 TWO_THIRD = 2.0/3.0
 
-# YIQ: used by composite video signals (linear combinations of RGB)
+# Clamping function to ensure we always return values in a valid range
```

Comment:
There is no need to have lower and upper parameters since they are not used.

```suggestion
# Clamp a value in the range [0.0; 1.0]
```

Reply:
Removed these parameters.